libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -203,13 +205,14 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _sem->give();
     }
 
+    // log raw data if flag is set
```

Comment:
I added that comment because it seemed to me that the implementation obfuscated the dependence on the flag by hiding it inside the one-line get_dataflash method.  Why is that preferable to testing _imu._log_raw_data at the point of doing the actual logging? I assume there's a good reason for it, but it cost me extra time in understanding how the code worked.